scripts/make-env

```diff
@@ -16,6 +16,8 @@ usage() {
   echo "  VPN_CRT    Path to the VPN server certificate"
   echo "  VPN_KEY    Path to the VPN server private key"
   echo "  VPN_DH     Path to the VPN server Diffie Hellman parameters"
+  echo "  SUPERUSER_EMAIL     Email address of the SuperUser"
```

Comment:
I think plain all-lowercase "superuser" is better.

Reply:
+1, I missed this but I can't un-see it now 😆